### ISSUE TITLE
clean up of the branches on the main repository

### ISSUE THREAD

**Issue by keewis** [pydata/xarray#4062]
We have several branches on our main repository that appear to be either out-dated or used to build documentation on RTD (these are no longer necessary since RTD fixed the `conda`-out-of-memory issues for everyone: building on personal RTD setups should work). Since they already lead to confusion, I think we should try clean them up. Here's a list of the branches I think we can remove:
- [x] `0.11.x`: seems to be for the `0.11` releases which are no longer maintained (points to the same commit as the `v0.11.3` tag)
- [ ] `accessor-documentation`: I intended to use this for building the documentation on RTD, but I accidentally opened #3988 from it instead of from the branch on my own repository. It will be removed once that PR is merged / closed.
- [x] `fix-docs` is from more than a year ago. @shoyer, is there anything that was not already merged into master?
- [x] `fix-rtd` was a attempt to fix RTD when it continuously failed about 1-2 months ago
- [x] `revert-3128-drop-keyword-support`: I think that was used to give a contributor the possibility to amend to a already merged PR, but in the end a follow-up PR was used instead.
- [x] `scipy19-docs` and `scipy19-docs-backup`: these were merged last year

Edit: I'm going to delete those branches on Monday

**Comment by keewis** [comment]
I removed all of them except `fix-docs`, which has 3 commits that are not marked with a PR number. After investigating, I think they are from:
- #2709
- #2708
- #2707

so I'll delete `fix-docs`, too.

**Comment by keewis** [comment]
now that every branch except `accessor-documentation` is gone, I'd say this is cleaned up

**Comment by max-sixty** [comment]
Thanks @keewis ! Tab-completes in git will be easier now!
